packages/lodestar/src/metrics/metrics/lodestar.ts

```diff
@@ -514,6 +514,11 @@ export function createLodestarMetrics(
 
     // Gossip block
     gossipBlock: {
+      elappsedTimeTillBecomeHead: register.histogram({
+        name: "lodestar_gossip_block_elappsed_time_till_become_head",
+        help: "Time elappsed between block slot time and the time block becomes head",
+        buckets: [0.1, 1, 10],
```

Comment:
It's very unlikely for a block to come exactly between 0 and 0.1 seconds, better bucket values would be:
```
[0.5, 1, 2, 4, 6, 12]
```